src/ui/components/FilterSwitch.tsx

```diff
@@ -8,11 +8,12 @@ import { ACTIVE_OPACITY } from 'ui/theme/colors'
 interface Props {
   toggle: () => void
   active: boolean
+  disabled?: boolean
```

Comment:
j'ai rajouté la fonctionnalité d'avoir un switch disabled, parce que je trouvais ça bizarre d'avoir un switch qui effectivement ne fait rien (cf return dans le toggle si not loggedin), donc je les ai disabled. Voir screenshot de la PR